### PR TITLE
Tao 650 xinclude model

### PR DESCRIPTION
**Pull request by ssipasseuth**

### CODE REVIEW

**Comment by ssipasseuth** [comment]
This requires https://github.com/oat-sa/tao-core/pull/291

File: views/js/qtiItem/test/xincludeLoader/test.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+require([
```
Comment (krampstudio) [inline views/js/qtiItem/test/xincludeLoader/test.js]: use `define`, not `require`

File: views/js/qtiItem/test/parser/test.js [hunk 2]
```diff
@@ -1,17 +1,53 @@
-require(['taoQtiItem/qtiItem/core/Parser', 'text!taoQtiItem/qtiItem/../../../test/samples/xml/qtiv2p1/associate.xml'], function(Parser, xml){
+require([
```
Comment (krampstudio) [inline views/js/qtiItem/test/parser/test.js]: use `define`, not `require`

**Comment by ssipasseuth** [comment]
PR updated

**Comment by ssipasseuth** [comment]
Related https://github.com/oat-sa/extension-tao-mediamanager/pull/15

**Comment by krampstudio** [comment]
See http://i.imgur.com/Ngcb1wS.png once the text block has a stimulus

**Comment by ssipasseuth** [comment]
Ok, you mean we can no longer select anything ? 
It has been planned (inside my head) to add some helpers to insert text or at least to set focus before or after such a big media box ?

Have you found other issues ?

**Comment by krampstudio** [comment]
Yes we can no longer select anything and the `A`  that disappear is confusing (I though there was an error)

**Comment by krampstudio** [comment]
I'll merge these PR and you can add fixes as they come

**Comment by ssipasseuth** [comment]
Ok, thanks. I will address them asap.

**Pull request merged**